terraform/environments/mlra/main.tf

```diff
@@ -1,90 +1,38 @@
-module "lb-access-logs-enabled" {
-  source = "github.com/ministryofjustice/modernisation-platform-terraform-loadbalancer"
+module "alb" {
+  # source = "github.com/ministryofjustice/modernisation-platform-terraform-loadbalancer"
+  source = "./modules/ALB"
   providers = {
     aws.bucket-replication = aws
   }
 
   vpc_all                    = local.vpc_all
   application_name           = local.application_name
   public_subnets             = [data.aws_subnet.public_subnets_a.id, data.aws_subnet.public_subnets_b.id, data.aws_subnet.public_subnets_c.id]
-  loadbalancer_egress_rules  = local.loadbalancer_egress_rules
-  loadbalancer_ingress_rules = local.loadbalancer_ingress_rules
+  # loadbalancer_egress_rules  = local.loadbalancer_egress_rules
+  # loadbalancer_ingress_rules = local.loadbalancer_ingress_rules
   tags                       = local.tags
   account_number             = local.environment_management.account_ids[terraform.workspace]
   region                     = local.application_data.accounts[local.environment].region
   enable_deletion_protection = false
   idle_timeout               = 60
   force_destroy_bucket       = true
+  ingress_cidr_block         = data.aws_vpc.shared.cidr_block
+  # egress_cidr_block          = ["0.0.0.0/0"]
+  deregistration_delay       = 30
+  healthcheck_interval       = 15
+  healthcheck_path           = "/mlra/"
```

Comment:
Now that we are referencing the health check path here, can we remove the variable that was previously created for it from environment_variables.json?

Reply:
are you referring to the file application_variables.json file:-
{
  "accounts": {
    "development": {
      "example_var": "dev-data",
      "region": "eu-west-2",
      **"alb_target_group_path": "/mlra/"**
    },

Reply:
Yes, exactly that: "alb_target_group_path": "/mlra/". No longer needed if we are taking it from the module call :)

Reply:
alb_target_group_path has been deleted from the file application_variables.json